src/libs/migrations/RenameReceiptFilename.ts

```diff
@@ -0,0 +1,56 @@
+import Onyx from 'react-native-onyx';
+import {NullishDeep, OnyxCollection} from 'react-native-onyx/lib/types';
+import Log from '@libs/Log';
+import ONYXKEYS from '@src/ONYXKEYS';
+import Transaction from '@src/types/onyx/Transaction';
+import {isEmptyObject} from '@src/types/utils/EmptyObject';
+
+type OldTransaction = Transaction & {receiptFilename?: string};
```

Comment:
Why not add it to `Transaction` type?

Reply:
This is a migration file, `receiptFilename` should be no longer used. I didn't want to add it to `Transaction` type to make sure nobody will use it in the future